### DISCUSSION TITLE
[Serenity Report] Log only request that fulfilled the criteria in Serenity Report

### DISCUSSION THREAD

**Discussion by akshay-klevu** [serenity-bdd/serenity-core#3587] [Q&A]
I have a been working on a scenario in which I make API calls using the wait mechanism like below : 
-  Make API call every 20 seconds and max time to retry is 2 minutes
- Another logic is there to check if the expected result is met is not we retry
- Now when I check the Report, all the retry calls are also available there. (Attached is screenshot showing the same)
- I don't want that, just log the request that gave the correct response
- is this achievable ??
- If YES then how should I proceed ? 
![Screenshot from 2024-12-12 13-20-55](https://github.com/user-attachments/assets/2dee2b95-c2c5-48d1-9555-b3b338aa3b68)

**Comment by wakaleo** [comment]
I would probably use plain RestAssured, not SerenityRest, for those queries.
